www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -821,41 +820,6 @@ function (declare, dom, style, on, query, Memory, Observable, topic, entities,
         }
       }));
 
-      //--- Details ---//
-
-      this.addChild(new Button({
-        label : 'Details',
```

Comment:
Ah, this is that ugly window of #566. Good riddance, I think. I'm okay with losing this information, didn't seem too useful in the first place. If analysing a file failed, where could have the user click `Details` to find the failure output and such in the first place?